.github/workflows/docker-image.yml

```diff
@@ -11,9 +11,12 @@ jobs:
     strategy:
       matrix:
         include:
-          - image: alpine315
-          - image: alpine316
-          - image: alpine317
+          - image: ubuntu1804
+          - image: ubuntu2004
+          - image: ubuntu2004-dpdk1911
+          - image: ubuntu2004-dpdk2011
+          - image: ubuntu2004-pfring
+          - image: ubuntu2004-zstd
```

Comment:
Don't we need to add centos7?

Reply:
The test linking failed because the project doesn't explicitly expose that we need to have c++11 to build examples

Reply:
Need this commit to be on master: https://github.com/seladb/PcapPlusPlus/pull/1075